src/commandHandlers/bio.ts

```diff
@@ -63,6 +63,8 @@ export const command = async (arg: string, embed: MessageEmbed, message: Message
             } catch (e) {
                 log.error(`ERROR: Couldn't get location ${data}`);
             }
+        } else if (field === 'twitter') {
+            data = `https://twitter.com/${data}`
```

Comment:
The `data` variable should be validated before appending it to twitter.com.
For example:
```typescript
if(data.trim().match(/^@?(\w{1,15})$/i)) {
    // ...
}
```



References:
- [Twitter - Username rules](https://help.twitter.com/en/managing-your-account/twitter-username-rules)
- [stackoverflow - Regular expression for twitter username](https://stackoverflow.com/questions/8650007/regular-expression-for-twitter-username)

Reply:
This is very true. I am happy for a new issue to be raised for this - that way we can keep PRs smaller but each still adding value.

Great suggestion @alrifay 👍

Reply:
How to create an issue that fixes something doesn't exist yet? 😁

Reply:
Maybe something like `twitter username validation` and reference this conversation in the PR

https://github.com/EddieJaoudeCommunity/EddieBot/pull/159#pullrequestreview-459677405

Reply:
Do you want me to implement validation in this PR?

Reply:
@AllanRegush No, I created a new issue for the validation as @eddiejaoude said. 😁